cmd/sync.go

```diff
@@ -71,6 +71,20 @@ func (raw *rawSyncCmdArgs) parsePatterns(pattern string) (cookedPatterns []strin
 	return
 }
 
+// it is assume that the given url has the SAS stripped, and safe to print
```

Comment:
At first I wondered how confident we are in this assumption, but then I remembered we redact all our GLCM and log outputs anway.